libflux/flux-core/src/semantic/nodes.rs

```diff
@@ -250,6 +250,16 @@ pub fn infer_pkg_types<T>(
 where
     T: Importer,
 {
+    if env.readwrite {
```

Comment:
Yes, though I count is a a bug since we are also the callers of this function (since the go tests fail there is still some issues around this so I guess it was good that I tried adding this).

Reply:
Do you think you can post the Go error? I'd like to know the error as I've suspected the Go code is wrong for some time, but have never known how it is wrong.